hw/hw11/mailroom.py

```diff
@@ -0,0 +1,120 @@
+if __name__ == '__main__':
```

Comment:
This is not how you want to use the `if __name__ == '__main__':` statement. You usually want to have this at the bottom of your script, and only include what you want to happen if you run this script as a stand-alone program. You would probably want to do this:

```
if __name__ == '__main__':
    start()
```

You want your module to be callable from other modules and adhere to the principals of [SOLID object-oriented design.](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)). Right now your whole script is contained within a giant if statement that says you can't run your script unless `__name__ == '__main__'`